src/ngraph/builder/norm.cpp

```diff
@@ -0,0 +1,153 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "norm.hpp"
+#include "ngraph/op/abs.hpp"
+#include "ngraph/op/add.hpp"
+#include "ngraph/op/constant.hpp"
+#include "ngraph/op/convert.hpp"
+#include "ngraph/op/multiply.hpp"
+#include "ngraph/op/not_equal.hpp"
+#include "ngraph/op/power.hpp"
+#include "ngraph/op/sqrt.hpp"
+#include "ngraph/op/sum.hpp"
+#include "ngraph/shape.hpp"
+
+using namespace std;
+
+namespace ngraph
+{
+    namespace builder
+    {
+        namespace detail
+        {
+            ///
+            /// \brief      Creates constant node with the given bias value.
+            ///
+            /// \param[in]  element_type  The node element type.
+            /// \param[in]  node_shape    The node shape.
+            /// \param[in]  bias          The value to fill node with.
+            ///
+            /// \return     The created Constant node.
+            ///
+            inline shared_ptr<Node> get_bias_node(const element::Type& element_type,
```

Comment:
I'm still not convinced this is function is needed. Please consider removing it.

Since it creates a `Constant`, it should be renamed to reflect what it does, rather than what it's used for - change the name to `make_constant_node`